docs/troubleshooting.md

```diff
@@ -19,3 +19,18 @@ You recently downloaded or updated XCode.
 2. Choose `Legacy Build System`
 3. Remove `node_modules/react-native/third-party` and
    `node_modules/react-native/third-party-podspecs`
+
+## I get Android crashes when using the Reader SDK
```

Comment:
This should be more specific about when the crash occurs (when the crash is triggered, it crashes with a specific error, etc.) as there are likely _other_ reasons why Android may crash when building with Reader SDK and we want folks to be able to distinguish which use case matches the problem they are seeing.